### PR TITLE
Bug 1469881 - Patches posted by Phabricator to Bugzilla don't list the patch author

### PR DESCRIPTION
**Pull request by dklawren**

### CODE REVIEW

File: extensions/PhabBugz/lib/Util.pm [hunk 1]
```diff
@@ -59,6 +59,14 @@ sub create_revision_attachment {
         ($timestamp) = Bugzilla->dbh->selectrow_array("SELECT NOW()");
     }
 
+    # If submitter, then switch to that user when creating attachment
+    my $old_user;
```
Comment (dylanwh) [inline extensions/PhabBugz/lib/Util.pm]:
```perl
try {

File: extensions/PhabBugz/lib/Util.pm [hunk 2]
```diff
@@ -76,6 +84,8 @@ sub create_revision_attachment {
     $bug->add_comment($revision->summary, { type       => CMT_ATTACHMENT_CREATED,
                                             extra_data => $attachment->id });
```
Comment (dylanwh) [inline extensions/PhabBugz/lib/Util.pm]:
```perl
} catch {
   die $_;
} finally {
```

File: extensions/PhabBugz/lib/Util.pm [hunk 3]
```diff
@@ -76,6 +84,8 @@ sub create_revision_attachment {
...
 
+    Bugzilla->set_user($old_user) if $old_user;
+
```
Comment (dylanwh) [inline extensions/PhabBugz/lib/Util.pm]:
```perl
};
```

**Comment by dklawren** [comment]
It will be tested by phabricator QA process. Yes.

**Pull request merged**